taxonomies/protocol-types.ttl

```diff
@@ -0,0 +1,34 @@
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix protocol-types: <http://bbp.epfl.ch/neurosciencegraph/taxonomies/protocoltypes> .
+
+
+protocol-types: a skos:ConceptScheme ;
+    skos:hasTopConcept protocol-types:activity.
+
+protocol-types:acquisitionannotation a skos:Concept ;
+    skos:broader protocol-types:activity;
+    skos:inScheme protocol-types: .
+
+protocol-types:reconstruction a skos:Concept ;
+    skos:broader protocol-types:activity;
+    skos:inScheme protocol-types: .
+
+protocol-types:brainslicing a skos:Concept ;
+    skos:broader protocol-types:activity;
+    skos:inScheme protocol-types: .
+
+protocol-types:stimulusexperiment a skos:Concept, nsg:ProtocolType;
+    skos:broader protocol-types:activity;
+    skos:inScheme protocol-types: .
+
+protocol-types:fixationstainingmounting a skos:Concept ;
+    skos:broader protocol-types:activity;
+    skos:inScheme protocol-types: .
+
+protocol-types:activity a skos:Concept, nsg:ProtocolType ;
+    skos:topConceptOf protocol-types: ;
+    skos:inScheme protocol-types: .
```

Comment:
This one is missing the wholecellpatchclamp activity

Reply:
Yes. This is the first version. We will add more in a next iteration.